apmpackage/apm/changelog.yml

```
@@ -37,13 +37,13 @@
     - description: add java_attacher support
       type: enhancement
       link: https://github.com/elastic/apm-server/pull/6741
-    - description: auth.anonymous.rate_limit.{event_limit,ip_limit} defaults are now the same as in the default rate_limit struct.
+    - description: `auth.anonymous.rate_limit.{event_limit,ip_limit}` defaults are now the same as in the default rate_limit struct.
```

Apparently you can't start a line with a backtick

```suggestion
    - description: the `auth.anonymous.rate_limit.{event_limit,ip_limit}` defaults are now the same as in the default rate_limit struct.
```

---

I thought this change would fix the error, but it didn't. Any ideas?

```
[2022-03-04T19:32:08.255Z] Error: checking package failed: formatting the integration failed (path: /var/lib/jenkins/workspace/pm-server_apm-server-mbp_PR-7467/src/github.com/elastic/apm-server/build/apmpackage, failFast: true): walking through the integration files failed: formatting file failed (path: /var/lib/jenkins/workspace/pm-server_apm-server-mbp_PR-7467/src/github.com/elastic/apm-server/build/apmpackage/changelog.yml): formatting file content failed: unmarshalling YAML file failed: yaml: line 40: found character that cannot start any token
```